### PR TITLE
Add pull request pipelines timeout

### PR DESCRIPTION
**Pull request by vsebe**
Set a 10 hours timeout for the entire pipeline build

[ci skip]

Signed-off-by: Violeta Sebe <vsebe@ca.ibm.com>

### CODE REVIEW

**Comment by vsebe** [comment]
@AdamBrousseau please review

File: buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All [hunk 1]
```diff
@@ -38,43 +38,45 @@ def OMR_BRANCH = 'openj9'
...
-    }
+            SPECS.each { SPEC ->
+                builds["build_JDK${SDK_VERSION} ${SPEC}"] = { build(OPENJDK_REPO, OPENJDK_BRANCH, OPENJDK_SHA, OPENJ9_REPO, OPENJ9_BRANCH, OPENJ9_SHA, OMR_REPO, OMR_BRANCH, OMR_SHA, SDK_VERSION, SPEC) }
+             }
+        }
 
-    cleanWs()
+        cleanWs()
+    }
+
```
[the remark this answers is not in the document]
Comment (vsebe) [reply buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All]: removed

File: buildenv/jenkins/jobs/pull-requests/PullRequest-Compile-JDK8-aix_ppc-64_cmprssptrs [hunk 2]
```diff
@@ -23,17 +23,18 @@
...
+            variableFile = load 'buildenv/jenkins/common/variables-functions'
+            variableFile.set_job_variables('build')
+            buildfile = load 'buildenv/jenkins/common/build'
+            cleanWs()
+        }
+    }
 
-    buildfile.build_all()
+    node("${NODE}") {
+        buildfile.build_all()
```
[the remark this answers is not in the document]
Comment (vsebe) [reply buildenv/jenkins/jobs/pull-requests/PullRequest-Compile-JDK8-aix_ppc-64_cmprssptrs]: re-added new line

File: buildenv/jenkins/jobs/pull-requests/PullRequest-Sanity-JDK9-linux_390-64_cmprssptrs [hunk 3]
```diff
@@ -24,28 +24,30 @@ SDK_VERSION = '9'
 SPEC = 'linux_390-64_cmprssptrs'
 TEST_TARGET = '_sanity'
 
+timeout(time: 10, unit: 'HOURS') {
 node('master') {
```
[the remark this answers is not in the document]
Comment (vsebe) [reply buildenv/jenkins/jobs/pull-requests/PullRequest-Sanity-JDK9-linux_390-64_cmprssptrs]: indented

**Comment by AdamBrousseau** [comment]
jenkins compile aix jdk9

**Comment by vsebe** [comment]
@pshipton please review, thank you!

**Pull request merged**